pkg/detectors/maxmindlicense/v2/maxmindlicense_v2.go

```diff
@@ -57,9 +57,7 @@ func (s Scanner) FromData(ctx context.Context, verify bool, data []byte) (result
 			req, err := http.NewRequestWithContext(
 				ctx, "POST", "https://secret-scanning.maxmind.com/secrets/validate-license-key",
 				strings.NewReader(data.Encode()))
-			if err != nil {
-				continue
-			}
+			r.SetVerificationError(err, keyRes)
```

Comment:
I think if there was an `err` that we wouldn't want to try use the `req` later. So I think I mislead you about using `SetVerificationError`

Reply:
I've added it. Is that what you meant ?